Client/Frontend/Reader/ReaderMode.swift

```diff
@@ -5,6 +5,15 @@
 import Foundation
 import WebKit
 
+enum ReaderModeMessageType: String {
+    case StateChange = "ReaderModeStateChange"
+    case PageEvent = "ReaderPageEvent"
+}
+
+enum ReaderPageEvent: String {
+    case PageShow = "PageShow"
```

Comment:
Are you planning on including additional events here? If not, having an enum for a single value seems odd.